src/pages/admin/queries.ts

```diff
@@ -0,0 +1,85 @@
+export const listCustomPlaylists = `
```

Comment:
Can we include this in `.graphqlconfig.yml` so we get type generation?

Reply:
Isn't there an issue with conflicting query names?